considerations.mkd

```diff
@@ -13,6 +13,24 @@
 
 # Interoperability Considerations
 
+## I-JSON
+
+GeoJSON texts SHOULD follow the constraints of I-JSON [RFC7493] for
+maximum interoperability.
+
+## Coordinate Precision
+
+The size of a GeoJSON text in bytes is a major interoperability
+consideration and precision of coordinate values has a large impact on
+the size of texts.  A GeoJSON text containing many detailed polygons can
+be inflated almost by a factor of two by increasing coordinate precision
+from 6 to 15 decimal places. 6 decimal places (a default common in,
```

Comment:
Do you want to say "For geographic coordinates, 6 decimal places ..." (since the precision depends on units)?